backend/app/services/memory/client.py

```
@@ -91,16 +106,52 @@ async def _get_session(self) -> aiohttp.ClientSession:
+                        and self._session_loop_id != current_loop_id
+                    )
+                )
+
+                if needs_new_session:
+                    # Close old session if it exists and is from a different loop
+                    if self._session and not self._session.closed:
+                        try:
+                            await self._session.close()
+                        except Exception:
+                            # Ignore close errors for sessions from different loops
+                            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)